### DISCUSSION TITLE
github Unavailable for your site because your domain is not properly configured to support HTTPS

### DISCUSSION THREAD

**Discussion by Fatih5252** [community/community#124615] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey! i having some issues with github pages. i cant activate the HTTPS option becuase it says that i did not configured my domain to support HTTPS can maybe anyone help?
![image](https://github.com/community/community/assets/154845930/5b93b91a-82c5-40e0-80e5-0e35a2724132)

here is my github DNS (i use IONOS btw)
![image](https://github.com/community/community/assets/154845930/04695cd0-2338-45d3-8ca4-21b019fc6966)
![image](https://github.com/community/community/assets/154845930/03034587-ff89-4784-ac25-3b42c7cb5243)

**Comment by gitstua** [comment, marked as answer]
Not sure if you are still working on this but a few ideas:
- make sure you have the SSL certificate bound to port 443 for the host you are using `www.fatih-simsek.de` or `fatih-simsek.de`
- check your SSL using e.g. https://www.ssllabs.com/ssltest/analyze.html?d=www.fatih-simsek.de
- do you need both A and CNAME records?
- should you be entering `www.fatih-simsek.de` as the custom domain or `fatih-simsek.de`

**Reply by Fatih5252** [reply]
thanks!

**Comment by gitstua** [comment]
Another thought - you should create a repo name of `fatih5252.github.io` as per https://docs.github.com/en/pages/quickstart#creating-your-website rather than https://github.com/Fatih5252/fatih-website

You can [rename a repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) if you like

**Reply by Fatih5252** [reply]
thanks!
